fuel-core-interfaces/src/info.rs

```diff
@@ -1,7 +1,10 @@
 use std::ops::Deref;
 
-use crate::types::ArcTx;
 use chrono::{DateTime, Utc};
+use fuel_tx::Transaction;
+use std::sync::Arc;
+
+pub type ArcTx = Arc<Transaction>;
 
 #[derive(Debug, Clone)]
 pub struct TxInfo {
```

Comment:
nit: I would just move this file to `fuel-core-interfaces/src/model/info.rs` and maybe even rename it to `txpool.rs` it feels like modules specific to txpool, that is why a little bit a specific naming of file.
other than that lgtm :)

Reply:
Ok, I wasn't sure what name so I chose `info` but I agree`txpool` is better to make it more clear in what it does and easier to find. It is now moved it to `fuel-core-interfaces/src/model/txpool.rs`